addons/account/tests/test_chart_template.py

```diff
@@ -359,15 +359,6 @@ def test_update_taxes_foreign_taxes(self):
         ])
         self.assertEqual(len(tax_test_model_data), 1, "Taxes should have been created even if the chart_template is installed through fiscal position system.")
 
-    def test_update_taxes_chart_template_country_check(self):
-        """ We can't update taxes that don't match the chart_template's country. """
-        self.company.chart_template_id.country_id = self.env.ref('base.lu')
-        # We provoke one recreation and one update
-        self.tax_template_1.amount += 1
-        self.tax_template_2.invoice_repartition_line_ids.tag_ids.name = 'tag_name_2_modified'
-        with self.assertRaises(ValidationError):
-            update_taxes_from_templates(self.env.cr, self.chart_template_xmlid)
-
```

Comment:
This might need some more adaptation.

Reply:
Indeed, is it OK to completely remove the test or should it be adapted?

Reply:
I discussed it a bit with CLBR and she told me that test was to prevent the system to update taxes that would have been used completely differently by our users. (for example, in previous version, tax hadn't any country.
During the upgrade process, it seems it's setting the fiscal country of the company on it.

Benj had concerns updating those tax.
I understand but the constraint that raises the error AND the test don't exist in the next versions anyway. :man_shrugging: 

I gave him a link to this conversation and ask him for the next steps to take toward this.
Either I just delete the test or I change something in the update process to make it fail.

Reply:
To me, it can be removed if we decided to backport the multivat feature. The test is a reliquate of former versions that is neither necessary nor correct functionally speaking since we know a tax can need tax grids from different countries (OSS & multivat namely). Why would we prevent this tax to be updated if the user wants to update its fiscal package. We know the +47 tax grid is sometimes missing on OSS / MultiVAT taxes. Removing the constraint and the test will help users have a correct taxes configuration easier.